File: app/src/main/java/fobo66/exchangecourcesbelarus/util/comparators/RURBuyComparator.java
package fobo66.exchangecourcesbelarus.util.comparators;

import fobo66.exchangecourcesbelarus.models.Currency;

/**
 * Created by fobo66 on 31.03.2016.
 */
public class RURBuyComparator implements CurrencyComparator {
    @Override
    public int compare(Currency lhs, Currency rhs) {
        return rhs.rurBuy.compareToIgnoreCase(lhs.rurBuy);
    }
}

File: app/src/main/java/fobo66/exchangecourcesbelarus/util/comparators/RurBuyComparator.kt
package fobo66.exchangecourcesbelarus.util.comparators

import fobo66.exchangecourcesbelarus.entities.Currency

/**
 * Created by fobo66 on 31.03.2016.
 */
class RurBuyComparator : CurrencyComparator {
  override fun compare(lhs: Currency, rhs: Currency): Int {
    return rhs.rurBuy.compareTo(lhs.rurBuy, ignoreCase = true)
  }
}
